components/datetime/src/lib.rs

```diff
@@ -137,26 +137,14 @@ pub mod preferences {
 }
 
 /// Types that can be fed to [`DateTimeFormatter`]/[`FixedCalendarDateTimeFormatter`].
+///
+/// This module contains reexports from the [`icu_calendar`] and [`icu_time`] crates.
```

Comment:
Nit, here and elsewhere:

```suggestion
/// This module contains re-exports from the [`icu_calendar`] and [`icu_time`] crates.
```